src/server/debug.rs

```diff
@@ -344,6 +344,28 @@ impl Debugger {
         Ok(errors)
     }
 
+    pub fn set_region_tombstone_by_id(&self, regions: Vec<u64>) {
+        let db = &self.engines.kv;
+        let wb = WriteBatch::new();
+        for region_id in regions {
+            let key = keys::region_state_key(region_id);
+            let region_state = db
+                .get_msg_cf::<RegionLocalState>(CF_RAFT, &key)
+                .unwrap()
+                .unwrap();
+            if region_state.get_state() == PeerState::Tombstone {
+                println!("skip because it's already tombstone");
+                return;
+            }
+            let region = &region_state.get_region();
+            write_peer_state(db, &wb, region, PeerState::Tombstone, None).unwrap();
```

Comment:
Do we need to write `MergeState`?

Reply:
It's not necessary because leave it in merging status is fine.